app/services/stock/package-lock.json

```diff
@@ -12,7 +12,9 @@
         "dotenv": "^8.2.0",
         "express": "^4.17.1",
         "jsonwebtoken": "^8.5.1",
-        "mongoose": "^5.11.15"
+        "mongoose": "^5.11.15",
+        "node-fetch": "^2.6.1",
+        "nodemon": "^2.0.7"
```

Comment:
Not really important but not sure why this diff is so large - I guess nodemon adds a lot?